src/content/ccip/tutorials/send-arbitrary-data-receipt-acknowledgment.mdx

```diff
@@ -0,0 +1,466 @@
+   enum MessageStatus {
+     NotSent, // 0
+     Sent, // 1
+     Confirmed // 2
+   }
+   ```
+
+1. When the transaction is marked with a "Success" status on the [CCIP Explorer](https://ccip.chain.link/), the CCIP transaction and the destination transaction are complete. The _acknowledger_ contract has received the message from the _message tracker_ contract.
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/ccip-explorer-sepolia-mumbai-success.webp"
+     alt="Chainlink CCIP Explorer - Sepolia to Mumbai Transaction Success"
+     style="max-width: 70%;"
+   />
+
+#### Acknowledgment message
+
+The `acknowledgePayLINK` function acts as a custom [`sendMessagePayLINK`](https://docs.chain.link/ccip/tutorials/send-arbitrary-data#sending-data-and-pay-in-link) function nested within the `ccipReceive` function of the _acknowledger_ contract and sends an acknowledgment (a CCIP message) to the _message tracker_ contract upon the initial message receipt. The data transferred in this acknowledgment message is the initial message ID.
+
+When the acknowledgment message is sent to the _message tracker_ contract, the _acknowledger_ contract emits an `AcknowledgmentSent` event.
+
+```solidity
+// Emitted when an acknowledgment message is successfully sent back to the sender contract.
+// This event signifies that the Acknowledger contract has recognized the receipt of an initial message
+// and has informed the original sender contract by sending an acknowledgment message,
+// including the original message ID.
+event AcknowledgmentSent(
+  bytes32 indexed messageId, // The unique ID of the CCIP message.
+  uint64 indexed destinationChainSelector, // The chain selector of the destination chain.
+  address indexed receiver, // The address of the receiver on the destination chain.
+  bytes32 data, // The data being sent back, usually containing the message ID of the original message to acknowledge its receipt.
+  address feeToken, // The token address used to pay CCIP fees for sending the acknowledgment.
+  uint256 fees // The fees paid for sending the acknowledgment message via CCIP.
+);
+```
+
+1. Copy your _acknowledger_ contract address from Remix. Open the [_Polygon Mumbai_ explorer](https://mumbai.polygonscan.com) and search for your deployed _acknowledger_ contract, then click on the **Events** tab.
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/polygon-mumbai-explorer-acknowledger-events.webp"
+     alt="Chainlink CCIP - Acknowledger Get Last Received Message Details"
+     style="max-width: 70%;"
+   />
+
+   The first indexed topic (`topic1`) in the `AcknowledgmentSent` event is the acknowledgment message ID sent to the _message tracker_ contract on _Ethereum Sepolia_. In this example, the message ID is `0x0defcd4d21415378e7136b7300940fb8c0e9e046002aa9b54a0362ecb5cee0ec`.
+
+1. Copy your own message ID from the indexed `topic1` and search for it in the [CCIP explorer](https://ccip.chain.link/).
+
+   <ClickToZoom
```

Comment:
This part will confuse readers. In fact, there is an event MessageReceived, but we cannot see any event AcknowledgmentSent.
At your place, I would have merged the 2 events (basically only keep AcknowledgmentSent and put relevant info of MessageReceived there). Thus, readers will only see one event